### PR TITLE
Fix missing array index in staging patch file

### PR DESCRIPTION
**Pull request by mvandenburgh**

### CODE REVIEW

File: k8s/gitlab/staging/release.yaml [hunk 1]
```diff
@@ -71,7 +71,7 @@ data:
     - op: remove
       path: /spec/values/gitlab/gitlab-shell/service
     - op: replace
-      path: /spec/valuesFrom/name
+      path: /spec/valuesFrom/0/name
```
Comment (scottwittenburg) [inline k8s/gitlab/staging/release.yaml:74]: I'm happy to just approve this and see what happens, but is there a way to test the patch syntax locally to make sure it's correct?

**Review by scottwittenburg — APPROVED** [review]

File: k8s/gitlab/staging/release.yaml (hunk 1 above)
Comment (mvandenburgh) [reply k8s/gitlab/staging/release.yaml:74]: yes, you can run the json patching script manually with
`python3 scripts/gitops-patch.py k8s/gitlab/release.yaml k8s/gitlab/staging/release.yaml`

Comment (mvandenburgh) [reply k8s/gitlab/staging/release.yaml:74]: Maybe it would be worth having a CI check that runs that script on every staging patch

Comment (scottwittenburg) [reply k8s/gitlab/staging/release.yaml:74]: Yeah, that might make sense.

**Pull request merged**